test/spec/unit/core/targeting_spec.js

```diff
@@ -330,6 +330,73 @@ describe('targeting tests', function () {
       expect(logErrorStub.calledOnce).to.be.true;
     });
 
+    describe('targetingControls.alwaysIncludeDeals', function () {
+      let bid4;
+
+      let targetingWithAdnxsDeal = {
+        hb_deal_rubicon: '1234',
+        hb_deal: '1234',
+        hb_pb: '0.53',
+        hb_adid: '148018fe5e',
+        hb_bidder: 'rubicon',
+        foobar: '300x250',
+        hb_deal_appnexus: '1234',
+        hb_pb_appnexus: '0.53',
```

Comment:
why isn't this 0.1 to match bid4.cpm?  Also, I would find it less confusing to use different deal IDs for RP and APN

Reply:
Good catch. Let me fix